source/configure/environment-configuration-settings.rst

```diff
@@ -4190,3 +4190,107 @@ This setting isn't available in the System Console and can only be enabled in ``
++-----------------------------------------------+---------------------------------------------------------------------------+
+
+.. config:setting:: redis-cache-address
+  :displayname: Hostname of the Redis host (CacheSettings)
+  :systemconsole: N/A
+  :configjson: RedisAddress
+  :environment: MM_CACHESETTINGS_REDISADDRESS
+  :description: Specify the hostname of the Redis host.
+
+Redis address
+^^^^^^^^^^^^^^
+
++-----------------------------------------------+---------------------------------------------------------------------------+
+| The hostname of the Redis host.               | - System Config path: **N/A**                                             |
+|                                               | - ``config.json setting``: ``CacheSettings`` > ``RedisAddress,``          |
+| String input.                                 | - Environment variable: ``MM_CACHESETTINGS_REDISADDRESS``                 |
++-----------------------------------------------+---------------------------------------------------------------------------+
+
+.. config:setting:: redis-cache-password
+  :displayname: Password of the Redis host (CacheSettings)
+  :systemconsole: N/A
+  :configjson: RedisPassword
+  :environment: MM_CACHESETTINGS_REDISPASSWORD
+  :description: Specify the password of the Redis host.
+
+Redis password
+^^^^^^^^^^^^^^
+
++-----------------------------------------------+---------------------------------------------------------------------------+
+| The password of the Redis host.               | - System Config path: **N/A**                                             |
+|                                               | - ``config.json setting``: ``CacheSettings`` > ``RedisPassword,``         |
+| String input. Leave blank if there is no      | - Environment variable: ``MM_CACHESETTINGS_REDISPASSWORD``                |
+| password.                                     |                                                                           |
++-----------------------------------------------+---------------------------------------------------------------------------+
+
+.. config:setting:: redis-cache-database
+  :displayname: Database of the Redis host (CacheSettings)
+  :systemconsole: N/A
+  :configjson: RedisDB
+  :environment: MM_CACHESETTINGS_REDISDB
+  :description: Specify the databse of the Redis host. Typically set to 0.
+
+Redis database
+^^^^^^^^^^^^^^
+
++-----------------------------------------------+---------------------------------------------------------------------------+
+| The database of the Redis host.               | - System Config path: **N/A**                                             |
+|                                               | - ``config.json setting``: ``CacheSettings`` > ``RedisDB,``               |
+| Typically set to ``0``.                       | - Environment variable: ``MM_CACHESETTINGS_REDISDB``                      |
```

Comment:
@agnivade - what does a value here of ``0`` mean exactly?

Reply:
It is the database number. It is a zero-indexed number up to 15. Redis allows a maximum of 16 databases.